src/main/java/org/spongepowered/common/Sponge.java

```diff
@@ -74,6 +91,43 @@ public Sponge(Injector injector, Game game, Logger logger,
 
         this.plugin = checkNotNull(plugin, "plugin");
         this.minecraftPlugin = checkNotNull(minecraftPlugin, "minecraftPlugin");
+
```

Comment:
It's not a good idea to put this directly in the constructor. In SpongeVanilla for example the Sponge class will be constructed even before the `MinecraftServer` gets called so this will just cause issues if something references the Minecraft server here in any way. Better place this in something like a `load()` method so the implementation can control when this is called.